### PR TITLE
Blood fountain

### PR DESCRIPTION
**Pull request by tinaStremenova**
- Added fountain assets
- Added fountain into level 1

https://user-images.githubusercontent.com/7025546/193992208-804a4553-b381-49d9-810c-e2a39636ecfd.mov

### CODE REVIEW

**Review by raulbatista94 — APPROVED** [review]

**Review by robha141 — APPROVED** [review]
🦟

File: SpriteKit/MendeluSpriteKit/Game/SceneObjects/Fountain.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+// MARK: Private API
+private extension Fountain {
+    func setupFountain() {
+        zPosition = Layer.fountain
+    }
+
+    func setupAction() {
+        let timePerFrame: TimeInterval = 0.12
```
Comment (matejmolnar) [inline SpriteKit/MendeluSpriteKit/Game/SceneObjects/Fountain.swift]: This property seems unnecessary 🤔

**Review by matejmolnar — APPROVED** [review]
🩸

File: SpriteKit/MendeluSpriteKit/Game/SceneObjects/Fountain.swift (hunk 1 above)
Comment (tinaStremenova) [reply SpriteKit/MendeluSpriteKit/Game/SceneObjects/Fountain.swift]: Yeah, good catch. I originally thought I'll have more animations. Removing.

**Pull request merged**